bot/exts/help_channels/_cog.py

```diff
@@ -664,3 +82,42 @@ async def helpdm_command(
         else:
             await _caches.help_dm.delete(ctx.author.id)
         await ctx.send(f"{constants.Emojis.ok_hand} {ctx.author.mention} Help DMs {state_str}!")
+
+    @help_forum_group.command(name="title", root_aliases=("title",))
+    async def rename_help_post(self, ctx: commands.Context, *, title: str) -> None:
+        """Rename the help post to the provided title."""
+        if not _channel.is_help_forum_post(ctx.channel):
+            # Silently fail in channels other than help posts
+            return
+
+        if not await commands.has_any_role(constants.Roles.helpers).predicate(ctx):
+            # Silently fail for non-helpers
+            return
+
+        await ctx.channel.edit(name=title)
+
+    @commands.Cog.listener()
+    async def on_thread_create(self, thread: discord.Thread) -> None:
+        """Defer application of new post logic for posts the help forum to the _channel helper."""
+        if thread.parent_id == self.help_forum_channel_id:
+            await _channel.help_thread_opened(thread)
+
+    @commands.Cog.listener()
+    async def on_thread_update(self, before: discord.Thread, after: discord.Thread) -> None:
+        """Defer application archive logic for posts in the help forum to the _channel helper."""
+        if after.parent_id != self.help_forum_channel_id:
+            return
+        if not before.archived and after.archived:
+            await _channel.help_thread_archived(after)
+
+    @commands.Cog.listener()
+    async def on_raw_thread_delete(self, deleted_thread_event: discord.RawThreadDeleteEvent) -> None:
+        """Defer application of new post logic for posts the help forum to the _channel helper."""
+        if deleted_thread_event.parent_id == self.help_forum_channel_id:
+            await _channel.help_thread_deleted(deleted_thread_event)
+
+    @commands.Cog.listener()
+    async def on_message(self, message: discord.Message) -> None:
```

Comment:
This get called for every message in every channel, which would cause the owner_id to raise an attribute error. We should remove the if statement below and just return early on `is_help_forum_post` instead.

Reply:
https://github.com/python-discord/bot/pull/2318/commits/13ec29de71871cf9e4ae84aeb604931bffecac6d